dashboard/models.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Actually I was trying to do this right now and I realized that the APIs logically belong here, because they implement a cached interface to edX.
Now I could move only the models to `Profile`, but I am not sure it makes much sense.